src/main/java/edu/cornell/kfs/module/purap/document/web/struts/IWantDocumentAction.java

```diff
@@ -957,15 +957,20 @@ public ActionForward insertBONote(ActionMapping mapping, ActionForm form, HttpSe
         }
         
         // If the note text is blank, set the attachment description as the text. Otherwise, concatenate both to form the text.
-        if (StringUtils.isBlank(note.getNoteText())) {
-            note.setNoteText(iWantDocumentForm.getIWantDocument().getAttachmentDescription());
+        if (StringUtils.isBlank(iWantDocumentForm.getIWantDocument().getNoteTextOption())) {
```

Comment:
The comment above does not seam to reflect this new functionality. We should either update or remove it.

Reply:
I will remove it.